### ISSUE TITLE
[MSG] Intermediate state between inquiry initialization & inquiry availability

### ISSUE THREAD

**Issue by sarahscott** [artsy/metaphysics#681]
@mzikherman and @ashkan18 , this is something I've heard about but do not have a complete understanding of - from what I understand, there is a delay between when a user starts to make an inquiry and when the inquiry is fully available from impulse to allow the collector some time to input personal and collection details. This might look a bit awkward in the app because the inbox view will always be available and I assume the collector will want to see the conversation immediately after it's initialized.

Is this definitely the case and if so, should we think about some sort of loading state for a conversation (i.e. 'Your conversation has been started and will have more information shortly) or should we get rid of the delay entirely?

**Comment by sarahscott** [comment]
Also if the delay is actually super short and not worth spending a lot of time thinking about, please let me know! I haven't delved very deeply into this.

**Comment by ashkan18** [comment]
Yes, that's true, as far as i understand, this only happens on collector's first inquiry when we wait for them to complete their profile and then we send it. I _think_ [this](https://github.com/artsy/force/blob/8b9ee8ab9c38cbf679719edc0f88aaa27cb30b10/desktop/apps/artwork/lib/inquire.coffee#L9) is the code that does it on Force side. Any subsequent inquiry after your first one should be sent immediately. Just keep in mind that few systems are involved here so by _immediately_ i mean possibly ~3 seconds.

Since that longer 10min delay only happens for first inquiry, I wonder if we need to have special case where if it's your first inquiry we show intermediate state.
